src/shared/components/svg-icon.js

```diff
@@ -61,29 +80,18 @@ export default function SvgIcon({
 }
 
 SvgIcon.propTypes = {
-  /**
-   * The name of the icon to display.
-   *
-   * The name must match a name that has already been registered using the
-   * `registerIcons` function.
-   */
-  name: propTypes.string,
-
-  /** A CSS class to apply to the `<svg>` element. */
+  name: propTypes.string.isRequired,
   className: propTypes.string,
-
-  /** Apply a style allowing for inline display of icon wrapper */
   inline: propTypes.bool,
-
-  /** Optional title attribute to apply to the SVG's containing `span` */
   title: propTypes.string,
 };
 
 /**
  * Register icons for use with the `SvgIcon` component.
  *
  * @param {IconMap} icons - Object mapping icon names to SVG data.
- * @param {boolean} [options.reset] - If `true`, remove existing registered icons.
+ * @param {Object} options
+ *  @param {boolean} [options.reset] - If `true`, remove existing registered icons.
```

Comment:
Per a suggestion from Kyle in another thread, I've started indenting object properties (the `options.reset` line here) that are specified "inline" for readability.

Reply:
Yeah, that's good.